virttest/utils_sys.py

```diff
@@ -0,0 +1,39 @@
+"""
+Virtualization test utility functions.
+
+:copyright: 2008-2009 Red Hat Inc.
```

Comment:
Can you please update it to the latest? Others look good to me.

Reply:
Sure